### PR TITLE
258 update map key to show 0 50mw with white text

### LINKED ISSUE #258: Update map key to show color for 0-50MW with white text
## Detailed Description

The map key should show the deepest gray map color (I think this is OCF black?) for 0-50MW and 0-50 and 50-150 MW should should have white text so it's more readable. 

Current: 

Screenshot 2022-09-29 at 15 52 07

Suggested: 

Screenshot 2022-09-29 at 15 48 44

## Context
This will help users more easily read the predictions for regions with lower solar energy levels. You can find the relevant design prototypes [here](https://www.figma.com/file/5RmuNt9H2398bXDGmPoz6V/Developer-Handoff?node-id=0%3A1) in New Layout. 

## Possible Implementation
Update frontend code.

### PR DESCRIPTION
**Pull request by braddf**
# Pull Request

## Description

Added property in legend item objects to control textColor.

image
image

NB. also has `main` changes pulled in to update `development` branch.

Fixes #258 and #244

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by rachel-labri-tipton — APPROVED** [review]

File: apps/nowcasting-app/constant.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const API_PREFIX = "https://api-dev.nowcasting.io/v0";
+export const API_PREFIX = "https://api-pro.nowcasting.io/v0";
```
Comment (rachel-labri-tipton) [inline apps/nowcasting-app/constant.ts:1]: what's the api-pro, @peterdudfield?

**Comment by rachel-labri-tipton** [comment]
@braddf, I ran the code locally and everything looks good. i do have a question about useMemo, but I'll ask that in slack.

File: apps/nowcasting-app/constant.ts (hunk 1 above)
Comment (braddf) [reply apps/nowcasting-app/constant.ts:1]: Oh good point @rachel-labri-tipton, is there any reason this is not an env var if it's just switching between the dev/prod endpoints @peterdudfield?

Comment (peterdudfield) [reply apps/nowcasting-app/constant.ts:1]: we should use ENV VAR. and development should definately be dev. 
Ill make an issue for this

Comment (peterdudfield) [reply apps/nowcasting-app/constant.ts:1]: https://github.com/openclimatefix/nowcasting/issues/263

**Review by peterdudfield — APPROVED** [review]
Just change back to dev, then this looks good

**Pull request merged**